test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8642,8 +8656,8 @@ def alloc_resource(self, node, resources):
 
         # if there is a dedicated time, move the availaility time up to that
         # time as necessary
-        if self.scheduler:
-            scheduler = self.scheduler
+        if self.schedulers[self.dflt_sched_name]:
```

Comment:
It seems that we should be running a loop here and calling each scheduler's parse_dedicated_time() method and it will just set server.dedtime for now because there is only one scheduler in the list.
but going forward server.dedtime should move to either scheduler or become a list.